frontend/src/components/PsyListing/PsyListing.jsx

```diff
@@ -110,6 +167,11 @@ const PsyListing = () => {
 
       return true;
     });
+
+    if (coords && addressFilter === AROUND_ME) {
+      return filteredPsychologists.sort((psyA, psyB) => distance.comparator(psyA, psyB, coords));
```

Comment:
If this is slow, you can start by adding the distance and then sort by distance
Would be much more efficient because here you are calculating the distance a bunch of time for each psy

Reply:
Indeed. Even if this is not slow, it should be done :) 

- [ ] improve distance comparator